modules/end-user-guide/partials/ref_devfile-reference.adoc

```diff
@@ -83,7 +77,7 @@ Project source consists of two mandatory values: `type` and `location`.
 
 `github`:: Same as `git` but for projects hosted on link:https://github.com/[GitHub] only. Use `git` for projects that do not use GitHub-specific features.
 
-`zip`:: Projects with sources in a ZIP archive. Location points to a ZIP file.
+`zip`:: Projects with sources in a `ZIP` archive. Location points to a `ZIP` file.
```

Comment:
Use the suffix to avoid using zip or ZIP (registered trademarks. See IBM SG pg. 315.
```suggestion
`zip`:: Projects with sources in a `.zip` archive. Location points to a `.zip` file.
```